db/raw/add-hotel-plugin.sql

```diff
@@ -0,0 +1,1576 @@
+--
+-- PostgreSQL database dump
+--
+
+SET statement_timeout = 0;
+SET lock_timeout = 0;
+SET client_encoding = 'UTF8';
+SET standard_conforming_strings = on;
+SET check_function_bodies = false;
+SET client_min_messages = warning;
+
+--
+-- Name: sqitch; Type: SCHEMA; Schema: -; Owner: postgres
+--
+
+CREATE SCHEMA sqitch;
+
+
+ALTER SCHEMA sqitch OWNER TO postgres;
+
+--
+-- Name: SCHEMA sqitch; Type: COMMENT; Schema: -; Owner: postgres
+--
+
+COMMENT ON SCHEMA sqitch IS 'Sqitch database deployment metadata v1.0.';
+
+
+--
+-- Name: plpgsql; Type: EXTENSION; Schema: -; Owner: 
+--
+
+CREATE EXTENSION IF NOT EXISTS plpgsql WITH SCHEMA pg_catalog;
+
+
+--
+-- Name: EXTENSION plpgsql; Type: COMMENT; Schema: -; Owner: 
+--
+
+COMMENT ON EXTENSION plpgsql IS 'PL/pgSQL procedural language';
+
+
+SET search_path = public, pg_catalog;
+
+SET default_tablespace = '';
+
+SET default_with_oids = false;
+
+--
+-- Name: admin_account; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE admin_account (
+    id uuid NOT NULL,
+    attendee_id uuid NOT NULL,
+    hashed character varying DEFAULT ''::character varying NOT NULL,
+    access character varying DEFAULT ''::character varying NOT NULL
+);
+
+
+ALTER TABLE public.admin_account OWNER TO rams_db;
+
+--
+-- Name: approved_email; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE approved_email (
+    id uuid NOT NULL,
+    subject character varying DEFAULT ''::character varying NOT NULL
+);
+
+
+ALTER TABLE public.approved_email OWNER TO rams_db;
+
+--
+-- Name: arbitrary_charge; Type: TABLE; Schema: public; Owner: rams_db; Tablespace: 
+--
+
+CREATE TABLE arbitrary_charge (
+    id uuid NOT NULL,
+    amount integer NOT NULL,
+    what character varying DEFAULT ''::character varying NOT NULL,
+    "when" timestamp without time zone NOT NULL,
+    reg_station integer
+);
+
+
+ALTER TABLE public.arbitrary_charge OWNER TO rams_db;
+
```

Comment:
No, you don't need to update the migrations for those plugins unless they're editing each other's tables (which... they really shouldn't, haha). This is just the raw file.

That being said, the problems here make a good case for just not checking in these raw files. Originally I was imagining they'd make a great backup, but as this illustrates, they're not very helpful unless they're tied to a specific event. We can change the way we generate the migrations so that you create a base SQL file, then reset your database, then create the new SQL file to compare to it.